_episodes/03-creating-containers.md

```diff
@@ -151,17 +151,120 @@ CONTAINER ID        IMAGE               COMMAND             CREATED
 ~~~
 {: .output}
 
-### Removing containers
-Just as there is a `docker image rm` command for removing images, there is a `docker container rm` command for removing containers. We should need to use that command in this session, though.
+### Removing images
+
+If you need to reclaim space, you will need to remove image files.
+The images and their corresponding containers can start to take up a lot of disk space of you don't clean them up occasionally.
+On macOS and Windows, when you uninstall the overall Docker software, it should have the effect of removing all of your image files, although we have not explicitly tested this.
+
+If you want to explicitly remove a container image, you will need to find out details such as the "image ID" or name of the repository. For example say my laptop contained the following image.
+~~~
+$ docker image ls
+~~~
+{: .language-bash}
+~~~
+REPOSITORY             TAG                 IMAGE ID            CREATED             SIZE
+ubuntu                 trusty              dc4491992653        12 months ago       222MB
+hello-world            latest              fce289e99eb9        4 months ago        1.84kB
+~~~
+{: .output}
+
+We can try to remove remove the `hello-world` image with a `docker image rm` command that includes the repository name, like so but...:
+~~~
+$ docker image rm hello-world
+~~~
+{: .language-bash}
+~~~
+Error response from daemon: conflict: unable to remove repository reference "hello-world" (must force) - container e7d3b76b00f4 is using its referenced image fce289e99eb9
+~~~
+{: .output}
+We get this error because there are containers created that depend on this image.
+
+### What containers are running?
+As indicated by the error above there is still an existing container from this image.
+We can list the running containers by typing.
+~~~
+$ docker container ls
+~~~
+{: .language-bash}
+~~~
+CONTAINER ID        IMAGE               COMMAND             CREATED             STATUS              PORTS               NAMES
+~~~
+{: .output}
+
+The first version of the command is ortholognal to what we know from the unix shell but we might want to use the shorter version. For refernece, "ps" stands for “Process Status”.
+We should also notice that this command didn't return any containers because our containers all exited and stopped running after they ran.
```

Comment:
Avoiding reuse of "ran". Maybe this is slightly clearer?
```suggestion
We should also notice that this command didn't return any containers because our containers all exited and thus stopped running after they completed their work.
```